CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Command Generate - Detect hasOne, hasMany and belongsToMany SQL relationships.
 
 ### Changed
+- Technical - Do not lint uncommitted files.
```

Comment:
Since it occurs before the commit itself (pre-commit) maybe we could have written:

```suggestion
- Technical - Do not lint untracked, not staged and deleted files.
```

But maybe it's less clear. Choose the wording you prefer!